bin/benchpark

```diff
@@ -125,17 +125,19 @@ def benchpark_check_system(arg_str):
 
 def benchpark_setup(subparsers, actions_dict):
     create_parser = subparsers.add_parser(
-        "setup", help="Create a benchmark and prepare it to build/run"
+        "setup", help="Set up an experiment and prepare it to build/run"
     )
 
     create_parser.add_argument(
-        "benchmark", type=str, help="The benchmark/experiment to run"
+        "benchmark", type=str, help="The experiment (benchmark/ProgrammingModel) to run"
+    )
+    create_parser.add_argument(
+        "system", type=str, help="The system on which to run the experiment"
     )
-    create_parser.add_argument("system", type=str, help="The system we are on")
     create_parser.add_argument(
         "workspace_root",
         type=str,
-        help="Where to install packages, store results, etc. (should be empty/nonexistent)",
+        help="Where to install packages, store results, etc. (could be empty/nonexistent)",
```

Comment:
We might want to say here that Benchpark expects to manage this directory (I think the original intent of the comment in "()" was that users should not pick a `workspace_root` that exists and contains files not related to `benchpark`: it should be empty/nonexistent the first time they run `benchpark`).

Reply:
Agreed.